docs/releases/5.1.2.txt

```diff
@@ -12,3 +12,5 @@ Bugfixes
 * Fixed a regression in Django 5.1 that caused a crash when using the
   PostgreSQL lookup :lookup:`trigram_similar` on output fields from ``Concat``
   (:ticket:`35732`).
+* Fixed a regression in Django 5.1 that caused a crash of ``JSONObject()``
```

Comment:
Could you please add a blank line before this entry?
```suggestion

* Fixed a regression in Django 5.1 that caused a crash of ``JSONObject()``
```

Reply:
Fixed.